requirements-minimal.txt

```diff
@@ -15,3 +15,4 @@ roiextractors @ git+https://github.com/catalystneuro/roiextractors.git@c23e4a003
 lxml>=4.6.5
 scipy>=1.4.1
 click  # must be unspecified version to handle platform/pyVersion
+pyopenephys==1.1.2
```

Comment:
We generally try to use `>=` for "requirements-minimal.txt". Is there a reason we need to use this specific version?

Reply:
Nop, this can be changed.